app/components/calculator_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<main class="calc-grid">
+<div class="calc-grid">
```

Comment:
`main` should wrap the body of the page (not the footer & header or other content that repeats across pages, but basically everything else)